_posts/2024/2024-02-06-about-caching-in-eo.md

```diff
@@ -0,0 +1,237 @@
+---
+layout: post
+date: 2024-02-06
+title: "Build cache in EO and other build systems"
+author: Alekseeva Yana
+---
+
+
+## Introduction 
+In [EO](https://github.com/objectionary/eo) a caching is used to speed up program execution.
+While developing [EO](https://github.com/objectionary/eo) we found a caching 
+[error](https://github.com/objectionary/eo/issues/2790) in `eo-maven-plugin`
+for EO version `0.34.0`. The error occurred because the cache was searched for the needed file using 
```

Comment:
@Yanich96 
Please, [use active voice, instead of passive](https://ell.stackexchange.com/questions/87182/active-and-passive-voice):
```
The error occurred because the cache was searched for the needed file using 
 a comparison of compilation time and caching time.
```

> In many styles of writing, active voice is preferred over passive voice for clarity and easier reading.